addon/TelemetryRappor.jsm

```diff
@@ -140,14 +140,63 @@ function makePRNG(seed) {
 }
 
 /**
+ * 
+ * @param {constant} hashingFunction  - Function used to encode. Supported: nsICryptoHash.MD5 and nsICryptoHash.SHA256
+ * @param {string} value - Value to encode.
+ * @param {integer} filterSize - Size of the bloom filter.
+ * @param {integer} numHashFunctions - Number of hash functions.
+ * @param {integer} cohort - Cohort.
+ */
+function encode(hashingFunction, value, filterSize, numHashFunctions, cohort) {
+  switch (hashingFunction) {
```

Comment:
Let's simplify this:

```
if (hashingFunction == Ci.nsICryptoHash.MD5) {
  return encodeMD5(...);
}

return encodeSHA256(...);
```

Reply:
What if someone tries to use another hash function?

Reply:
The documentation clearly states that we only support these two. Moreover, we're the one executing the addon, so we should be covered.

Reply:
Done